skyuxconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "skyux-tree-view",
+  "$schema": "./node_modules/@skyux/config/skyuxconfig-schema.json",
+  "mode": "easy",
+  "compileMode": "aot",
+  "plugins": [
+    "@skyux-sdk/builder-plugin-skyux"
+  ],
+  "testSettings": {
+    "e2e": {
+      "browserSet": "speedy"
+    },
+    "unit": {
+      "browserSet": "paranoid"
+    }
+  },
+  "app": {
+    "styles": [
+      "src/app/public/scss/angular-tree-styles.scss"
```

Comment:
Will consumers need to pull this style sheet into their SPAs/libraries, too?

Reply:
Whoops, no. That was copy+pasta from ag-grids. I've removed this.